pkg/kubelet/image_manager.go

```diff
@@ -31,6 +31,10 @@ import (
 	"k8s.io/kubernetes/pkg/util/sets"
 )
 
+const (
+	defaultGCAge = time.Minute * 5
```

Comment:
This seems pretty long. Why would kubelet take 5 minutes between pulling an image and starting a container?
Can this be 1 minute? cc @yujuhong